install/0000_90_cluster-version-operator_02_servicemonitor.yaml

```diff
@@ -40,19 +40,26 @@ spec:
       for: 10m
       labels:
         severity: critical
+    - alert: CannotRetrieveUpdates
+      annotations:
+        message: Cluster version operator has not retrieved updates since {{ "{{ $value | humanizeDuration }}" }}. Failure reason {{ "{{ $labels.reason }}" }}. Console {{ "{{ $labels.console_url.url }}" }}.
```

Comment:
This console URI still needs [a `/settings/cluster/` suffix][1] to get you from the default console page to the ClusterVersion page.

[1]: https://github.com/openshift/cluster-version-operator/pull/357#discussion_r428156247

Reply:
Ack. This was just trying to figure out how to pull the url from the metric.

Reply:
suffix added